### ISSUE TITLE
Missing Control Sensor /nzxt/krakenz

### ISSUE THREAD

**Issue by raeldor** [Rem0o/FanControl.Releases#2789]
This has been working great and suddenly in the last couple of weeks I have started getting this error.

<img width="494" alt="image" src="https://github.com/user-attachments/assets/51807352-f0e2-4426-9a5e-dcac2be5c2af">

Is this a known issue, or has something gone wrong with my NZXT hardware?  I haven't installed any NZXT updates as far as I remember.

**Comment by raeldor** [comment]
I went back to v197 and the message disappeared, but all my fan setup seems to have gone too, even though kraken is still mentioned in the userconfig file.

**Comment by Rem0o** [comment]
Does "suddenly" corresponds to an update for FanControl?  Did you go from 174 to 20X?

**Comment by raeldor** [comment]
No, I've been updating each time there is an update, so I would say it's in the last couple of updates or so that I've noticed is

**Comment by Rem0o** [comment]
Do yo use these at all in FanControl? If not, just save your config on top of the error and it will go away.

**Comment by raeldor** [comment]
Looking at the restored app and config it's paired to a sensor called CPU Pump.  If I search the config for this, then I see...

      {
        "Identifier": "/nzxt/krakenz/3359334B3538/fan/0",
        "IsHidden": false,
        "Name": "Pump",
        "NickName": "CPU Pump"
      },

So, I think yes, it was being used and now in the new version is no longer being recognized.

**Comment by Oyashirox** [comment]
I've been having the same issue 

![image](https://github.com/user-attachments/assets/fe320d19-8d07-43bc-b7c6-5a33178e8fcc)

Last working version for me is v201, it started appearing with v202

**Comment by raeldor** [comment]
Hi.  I installed the Kraken plug-in and then restored my old config file and it seems to be working again now.  In fact, I can now add the pump and fans and control them, which I couldn't do before.  I thought the Kraken was supported natively, but maybe something changed there?

**Comment by semool** [comment]
The Native Kraken Support in LibHardware is not good. I also tested it and have Problems with my Kraken 280 (2023).
Back to the Plugin and all is fine and running. Hope the Plugin gets Updates in the Future for the new Krakens.

**Comment by mmshivesh** [comment]
Encountering the same issue with V209. Downgraded back to V206 to fix it. There's definitely a regression here.

**Comment by Rem0o** [comment]
I believe the identifiers got modified, hence why the ones saved in your config are not recognized. Does a new "virgin" control card appeared?

**Comment by mmshivesh** [comment]
@Rem0o You are correct- I see a brand new set of AIO controls, which work fine after I configure them:

![image](https://github.com/user-attachments/assets/ca116753-e4c1-49bf-9ef5-18d8710278d7)

The "duplicate" older controls don't do anything even if I try to control it manually after pairing. Is there a way to remove these? I'm assuming FanControl is pulling these from the system as it looks like my motherboard reports the PUMP1 header separately from the Pump control also reported by NZXT:

![image](https://github.com/user-attachments/assets/dd9a0916-08e8-4924-a42e-67088c808189)

**Comment by Rem0o** [comment]
@mmshivesh  could you send your new json config file saved? Really confusing as to what is what.

**Comment by raeldor** [comment]
I re-did everything and it was working perfectly, now it's started failing again...
<img width="424" alt="image" src="https://github.com/user-attachments/assets/f34218c8-52e6-4730-93ce-a6f1c714e834">
